km52/Kholin_Nikita/laboratory1.sql

```diff
@@ -1,2 +1,114 @@
 -- LABORATORY WORK 1
 -- BY Kholin_Nikita
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+видаляти дані з таблиці
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+create user kholin identified by kholin_password;
+grant connect to kholin;
+grant delete on kholin.people to kholin;
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Людина має лікарняну картку, що містить записи про історію хвороби.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+create table people
+(
+  person_id char(10) not null,
+  name char(30) not null
+);
+
+create table cards
+(
+  card_id char(10) not null,
+  person_id char(10) not null
+);
+
+create table card_records
+(
+  card_record_id char(10) not null,
+  card_id char(10) not null
+);
+
```

Comment:
type should be not a "char", but "varchar2" and add two new line :
`card_record_id vsrchar2(10) not null,
 card_id varchar2(10) not null,
 person_id varchar2(10) not null,
 adress varchar2(30) not null`